src/main/resources/application.properties

```diff
@@ -2,8 +2,10 @@ spring.application.name=BookStore
 spring.datasource.url=jdbc:mysql://localhost/book_store?serverTimezone=UTC
 spring.datasource.driverClassName=com.mysql.cj.jdbc.Driver
 spring.datasource.username=root
-spring.datasource.password=pass
+spring.datasource.password=!Vladius080197
```

Comment:
It is unsafe to push passwords on git, you should left this field blank or use env variable